src/pages/Events/index.tsx

```diff
@@ -1,24 +1,28 @@
+import RHDevs from '../../assets/RHDevs-Logo-EventsPage.png'
+import UPGRHADEimg from '../../assets/UPGRHADE-Logo-EventsPage.png'
+import InterHallimg from '../../assets/InterHall-Logo-EventsPage.png'
 import PageSectionComponent from '../../components/PageSection'
 import { MainContainer } from './styles/EventPage.styled'
-import { interHallHackathon } from '../../texts/descriptions/events'
+import { Adesc, Atitle, Bdesc, Btitle, Cdesc, Ctitle } from '../../texts/descriptions/events'
 
 export default function Events() {
   return (
     <MainContainer>
       <PageSectionComponent
-        title={interHallHackathon.eventName}
-        description={interHallHackathon.eventDescription}
+        title={Atitle}
+        description={Adesc}
         hasImage
-        textPosition="left"
-        image=""
+        textPosition="right"
+        image={InterHallimg}
         events
       />
+      <PageSectionComponent title={Btitle} hasImage description={Bdesc} textPosition="left" image={RHDevs} events />
       <PageSectionComponent
-        title={interHallHackathon.eventName}
-        description={interHallHackathon.eventDescription}
+        title={Ctitle}
         hasImage
+        description={Cdesc}
         textPosition="right"
-        image=""
+        image={UPGRHADEimg}
         events
       />
```

Comment:
the method to store events is not optimised.

create a type event where you have title, desc and imgUrl. This will make scaling much simpler. Can discuss w the seniors on board.